### PR TITLE
Fix Shopping-cart CSS and JS

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

**Comment by stvstnfrd** [comment]
retest this please

**Comment by caesar2164** [comment]
[The shopping cart functionality that we now have on Lagunita does not work here.](https://trello.com/c/8h1qbRrA/147-the-shopping-cart-functionality-that-we-now-have-on-lagunita-does-not-work-here)

**Comment by stvstnfrd** [comment]
@caesar2164 Tests are getting a segfault; what happens locally?

**Comment by caesar2164** [comment]
Yeah, this is not working yet.

**Comment by stvstnfrd** [comment]
+1 pending tests

**Comment by caesar2164** [comment]
test this please

File: lms/templates/shoppingcart/shopping_cart.html [hunk 1]
```diff
@@ -409,8 +409,8 @@ <h3 class="course-title-info" id="course-title">
                           var oldId = data['oldToNewIdMap'][i]['oldId'];
                           var newId = data['oldToNewIdMap'][i]['newId'];
 
-                          $('input.spin-counter[data-item-id]=' + oldId ).data('item-id', newId);
-                          $('button.btn-remove[data-item-id]=' + oldId ).data('item-id', newId);
+                          $('input.spin-counter[data-item-id' + oldId + ']').data('item-id', newId);
```
Comment (stvstnfrd) [inline lms/templates/shoppingcart/shopping_cart.html]: Wait, where'd the `=` sign go?
Don't we need that?

**Pull request merged**